packages/manager/apps/iam/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@ovh-ux/iam-app",
```

```suggestion
  "name": "@ovh-ux/manager-iam-app",
```
To be consistent with other apps packages